themes/default/content/blog/retainondelete/index.md

```diff
@@ -0,0 +1,108 @@
+# A placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: meta.png
+
+# At least one author is required. The values in this list correspond with the `id`
+# properties of the team member files at /data/team/team. Create a file for yourself
+# if you don't already have one.
+authors:
+    - fraser-waters
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - features
+
+# See the blogging docs at https://github.com/pulumi/docs/blob/master/BLOGGING.md.
+# for additional details, and please remove these comments before submitting for review.
+---
+
+While Pulumi lets you manage your infrastrucutre with code there are some cases where operations need to
+happen outside the standard deployment lifecycle. Pulumi now supports retaining resources on delete allowing a
+number of new workflows.
+
+<!--more-->
+
+We've seen a number of user questions over the years about keeping resources after pulumi deployments, or
+sharing resources between deployments. All of these worked down to needing a new [resource
+option](https://github.com/pulumi/pulumi/issues/7747) to tell the Pulumi engine to not actually delete a
+resource when replacing or removing it from the stack.
+
+## Retain on delete
+
+We now support a new resource option [retain on delete]({{< relref "docs/intro/concepts/resources/options/retainOnDelete" >}})
+that allows you to specifiy this case to the engine.
+
+When a resource marked with `retainOnDelete` is deleted or replaced by a Pulumi update the engine will not
+call through to the resource providers `Delete` function. This is the similar to AWS Cloud Formation's [Retain
+DeletionPolicy](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-attribute-deletionpolicy.html).
+
+## Supporting data storage policies
+
+Using this new feature it is now possible to uncouple resource lifecycles from deployment cycles. This can be
+espically useful to teams with policy around data and storage. For example your company may require that all
+S3 storage is backed up and retained for some amount of time. Using `RetainOnDelete` you can ensure that the
+S3 buckets that Pulumi creates aren't deleted by Pulumi, leaving the data alive for your storage policy team
+to backup and then delete when no longer required.
+
+## Supporting shared resources
+
+Another workflow this option enables is the ability to manage externaly owned resources via Pulumi deploments.
+You may have an external long lived resource that you want to temporarily manage via a Pulumi program, but
+then relquinsh back to the external owner rather than deleting when your finished with it.
+
+Using a combiniation of [import](https://www.pulumi.com/docs/guides/adopting/import/) and `retainOnDelete` you
+can bring a resource under managment and then relinqish it back to the external owner.
+
+## Working around provider bugs
```

Comment:
Should we sugarcoat "bugs"? Sounds a bit harsh :)

Reply:
problems, issues, failures?